dataPipelines/gc_scrapy/gc_scrapy/utils.py

```diff
@@ -42,6 +44,18 @@ def dict_to_sha256_hex_digest(_dict: Dict[Any, Any]) -> str:
 
     return str_to_sha256_hex_digest(value_string)
 
+def get_pub_date(publication_date):
```

Comment:
we need to test to make sure this function works and does not default to "None" automatically

Reply:
We've tested for all crawlers that utilize this function as part of the "compare analysis" between the dev and matt-dev branches, and the function works as intended